src/component/feature/form/__tests__/__snapshots__/form-add-feature-component-test.jsx.snap

```diff
@@ -2,7 +2,6 @@
 
 exports[`render the create feature page 1`] = `
 ShallowWrapper {
```

Comment:
these snapshots are weird... Don't we have https://github.com/adriantoine/enzyme-to-json?